### ISSUE TITLE
collapsed view overlaps with bookmarks bar

### ISSUE THREAD

**Issue by michaelrhanson** [mozilla/socialapi-dev#3]
STR:

```
make sure your bookmarks bar has bookmarks all the way to the right
collapse the social sidebar
```

-> the social sidebar overlaps the bookmarks, as opposed to shoving the menu to the left.

**Comment by mhammond** [comment]
There are a variety of similar problems - eg, unselecting "tabs on top".  I guess one way to tackle this is that instead of hard-coding the nav-bar, we need to work out which child of navigator-toolbox is actually at the bottom and use that _and_ listen for events which tell us this has changed as the user customizes the UI.  I had a bit of a fiddle using the boxObject to tell us the actual order but can't make it work as documented yet.

**Comment by mhammond** [comment]
should also be fixed in rev 39afe348 - although see issue 18 for some caveats
